src/main/resources/base/console_backend.capnp

```diff
@@ -470,6 +470,7 @@ struct NtripConnect {
         pos @4 :Position;
         none @5 :Void;
     }
+    outputType @6: Text;
```

Comment:
Considering enum or just boolean `use_sbp` to save some negligible amount of parsing and capnproto allocations 🔨